Utilities/build-script-helper.py

```diff
@@ -190,8 +195,6 @@ def build_single_product(product: str, swift_exec: str, args: argparse.Namespace
     """
     swiftpm_args = get_swiftpm_options(swift_exec, args)
     additional_env = get_swiftpm_environment_variables(swift_exec, args)
-    if args.action == 'install':
-      additional_env['SOURCEKIT_LSP_CI_INSTALL'] = "1"
```

Comment:
I think I put this here and not in `get_swiftpm_environment_variables()` initially either because it broke something when [invoking `swiftpm_bin_path()` also in the install action](https://github.com/apple/sourcekit-lsp/blob/4a22e0e725733d52f76ae692b6115ba89d302afe/Utilities/build-script-helper.py#L228) or I was concerned it would, I forget which. Let's try it on the CI and find out.